registry/dbConfig/postgres/schemas/auth-basic.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE users (
+    "id" SERIAL,
+    "user_id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "username" VARCHAR(255) UNIQUE NOT NULL,
```

Comment:
If a engineer doesn't want usernames in their system and instead wants to use emails, how would they go about doing this? Would they need to rename this column? Just place emails in this column?...

Reply:
they can replace the column here or add it and modify the user repository and `authBasic` controller. We should have these kind of things in the documentation for each module I guess. Like guides of "How to implement common use cases" etc.